init-tools.ps1

```diff
@@ -57,6 +57,61 @@ if (!(Test-Path "$BUILD_TOOLS_PACKAGE_PATH\init-tools.cmd"))
     exit 1
 }
 
+# Bring down the CLI for build tools
+$DOTNET_PATH=$BUILD_TOOLS_PATH + "\dotnetcli"
```

Comment:
Yeah. All this is duplicate. We are already installing a CLI. Also, why not use the install script to obtain a CLI? That is, if we decide to keep two of these "stage0"s.